docs/develop/typescript-restrictions.md

```diff
@@ -57,6 +57,23 @@ The following words can't be used as identifiers in a script. They are reserved
 * `ExcelScript`
 * `console`
 
+## Only arrow functions in array callbacks
+
+Your scripts can only use [arrow functions](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Functions/Arrow_functions) when providing callback arguments for [Array](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array) methods. You cannot use any sort of identifier or "traditional" function in these methods.
```

Comment:
Ambiguous.

Would it be reasonable to say that you may not **pass** any sort of identifier or ...